src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.js

```diff
@@ -30,11 +30,28 @@ const IncidentSplitContainer = ({ FormComponent }) => {
   const dispatch = useDispatch();
   const [parentIncident, setParentIncident] = useState();
   const [directingDepartment, setDirectingDepartment] = useState([]);
+  const [note, setNote] = useState();
   const departments = useSelector(makeSelectDepartments);
   const directingDepartments = useSelector(makeSelectDirectingDepartments);
 
   const [subcategoryGroups, subcategoryOptions] = useSelector(makeSelectSubcategoriesGroupedByCategories);
 
+  const parentIncidentPatchData = useMemo(() => {
```

Comment:
I would use a useCallback call and not an useMemo here. What do you think @boris-arkenaar ?
Long names are difficult to read especially for component internal members. In this case `patchData` or `getPatchData` would be enough to understand what's going on.

Reply:
Will rename `parentIncidentPatchData ` to `patchData`, but I'm curious about the benefit of using useCallback instead of useMemo here.

Reply:
As it is, it is calculating data that is being used elsewhere. That means `useMemo` is the correct way to go.

You could argue that this logic can be executed below, directly where the data is actually being used. Since that's the only place it's being used, I believe. You might then be inclined to extract that logic from there to a separate function, to keep the code clean, and that function would be inside a `useCallback`. So, following that trail of thought, this can go inside a `useCallback` indeed, instead of a `useMemo`.